llvm/test/CodeGen/X86/GlobalISel/legalize-sdiv.mir

```diff
@@ -0,0 +1,176 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -mtriple=x86_64-linux-gnu -run-pass=legalizer -verify-machineinstrs %s -o - | FileCheck %s --check-prefixes=CHECK,X64
+# RUN: llc -mtriple=i686-linux-gnu -run-pass=legalizer -verify-machineinstrs %s -o - | FileCheck %s --check-prefixes=CHECK,X86
+
+--- |
+  source_filename = "sdiv.ll"
```

Comment:
Don't need the IR section

Reply:
I've always thought that IR section is needed to regenerate MIR if something has changed. Or is it not a common practice?

Reply:
No, it's needed for a specific set of IR references (block names, IR values in MMOs, and global references)